web/static/web/css/header.css

```diff
@@ -1,144 +1,332 @@
-.nav {
+#header {
+    --logo-padding: 30px;
+    --logo-width: 360px;
+    --logo-min-width: 120px;
+    --logo-space: minmax(var(--logo-min-width), var(--logo-width));
+
+    --burger-margin: 2em;
+
+    --header-height: 120px;
+
+    /* nav */
+    --nav-space-between: 2em;
+    --bubble-background-extra-width: calc(1.5 * var(--nav-space-between));
+    --dur: 0.15s;
+    --font-size: 1.25em;
+
     display: grid;
-    grid-template-columns: minmax(120px, 1fr) 3fr;
-    grid-template-rows: 120px;
+    grid-template-columns: 1fr auto 1fr;
     width: 100%;
-    height: 120px;
-    background-color: rgb(33, 43, 52);
+    height: var(--header-height);
+    background-color: rgb(34, 43, 52);
 }
 
-.nav .menu {
-    align-self: center;
-    margin: 0;
-    padding: 0 30px;
+#header .nav-item .nav-item-text,
+#nav-user-dropdown .dropdown-item .dropdown-item-text,
+.lang-link .item .text {
+    color: #DCDDDE;
 }
 
-.nav .logo {
+#header .nav-item:hover .nav-item-text,
+#nav-user-dropdown .dropdown-item:hover .dropdown-item-text,
+.lang-link .item:hover .text {
+    color: #FFF;
+}
+
+#header div.logo-centering {
+    display: none;
+}
+
+#header .logo {
+    display: grid;
+    grid-template-columns: var(--logo-padding) var(--logo-space) minmax(var(--nav-space-between), var(--logo-padding));
     align-self: center;
-    padding: 0 30px;
 }
 
-.nav .logo img {
+#header .logo img {
+    margin: auto;
+    max-height: 50px;
     object-fit: contain;
 }
 
-.limg {
+#header .logo .limg.large {
+    display: block;
+}
+
+#header .logo .limg.medium {
     display: none;
 }
 
-.limg.large {
-    display: inline;
+#header .logo .limg.small {
+    display: none;
 }
 
-.burger {
+#header #burger {
     display: none;
+    align-self: center;
+    justify-self: end;
+    margin-right: var(--burger-margin);
     margin-bottom: -10px;
 }
 
-.burger:hover {
+#header #burger:hover {
     cursor: pointer;
 }
 
-.burger {
-    align-self: center;
+#header #burger i {
+    color: #FFFFFF;
+    font-size: 30px;
+}
+
+#header #nav {
     justify-self: center;
+    font-size: var(--font-size);
 }
 
-.burger i {
-    color: #FFFFFF;
-    font-size: 30px;
+#header #nav .nav-item,
+#header #side-nav .nav-item {
+    height: calc(var(--font-size) + 2em);
+    margin: calc(var(--nav-space-between) / 2);
+    padding: 0;
+    border-radius: 0;
+    justify-content: center;
 }
 
-.nav .ui.inverted.menu .item {
-    color: rgba(255, 255, 255, .7) !important;
+#header #nav .nav-item:hover,
+#header #side-nav .nav-item:hover,
+#nav-user-dropdown.active {
+    /* Disables property from Semantic UI */
+    background-color: transparent;
 }
 
-.lang-form {
-    display: none;
+#header #nav .nav-item .bubble-background,
+#header #side-nav .nav-item .bubble-background {
+    position: absolute;
+    width: 100%;
+    height: var(--font-size);
+    border-radius: 2em;
+    z-index: 1;
```

Comment:
Is the z-index here needed? I would think that the only needed z-index is the one for the text, so that it overlaps the bubble.

Reply:
Oh, yes, that's true! It was a remnant from earlier in the development, when the bubble-backgrounds used to overlap each other.